packages/store/src/types.js

```diff
@@ -12,111 +12,105 @@
- */
+// TODO placeholder. Figure out how to import from `@endo/patterns` instead
+/** @typedef {(...args: any[]) => any} Method */
 
+// TODO placeholder. Figure out how to import from `@endo/patterns` instead
 /**
  * @typedef {{
- * klass: 'Interface',
- * interfaceName: string,
- * methodGuards: Record<string | symbol, MethodGuard>
- * sloppy?: boolean
+ *   klass: 'Interface';
+ *   interfaceName: string;
+ *   methodGuards: Record<string | symbol, MethodGuard>;
+ *   sloppy?: boolean;
  * }} InterfaceGuard
- * TODO placeholder. Figure out how to import from `@endo/patterns` instead
  */
 
-/**
- * @typedef {any} MethodGuardMaker
- * TODO placeholder. Figure out how to import from `@endo/patterns` instead
- */
+// TODO placeholder. Figure out how to import from `@endo/patterns` instead
+/** @typedef {any} MethodGuardMaker */
 
+// TODO placeholder. Figure out how to import from `@endo/patterns` instead
 /**
- * @typedef {{ klass: 'methodGuard', callKind: 'sync' | 'async', returnGuard: unknown }} MethodGuard
- * TODO placeholder. Figure out how to import from `@endo/patterns` instead
+ * @typedef {{
+ *   klass: 'methodGuard';
+ *   callKind: 'sync' | 'async';
+ *   returnGuard: unknown;
+ * }} MethodGuard
  */
 
-/**
- * @typedef {any} ArgGuard
- * TODO placeholder. Figure out how to import from `@endo/patterns` instead
- */
+// TODO placeholder. Figure out how to import from `@endo/patterns` instead
+/** @typedef {any} ArgGuard */
 
 // /////////////////////////////////////////////////////////////////////////////
 
 /**
- * @typedef {object} StoreOptions
- * Of the dimensions on which KeyedStores can differ, we only represent a few
- * of them as standard options. A given store maker should document which
- * options it supports, as well as its positions on dimensions for which it
- * does not support options.
- * @property {boolean} [longLived=true] Which way to optimize a weak store. True means
- * that we expect this weak store to outlive most of its keys, in which
- * case we internally may use a JavaScript `WeakMap`. Otherwise we internally
- * may use a JavaScript `Map`.
- * Defaults to true, so please mark short lived stores explicitly.
- * @property {boolean} [durable=false]  The contents of this store survive termination
- *   of its containing process, allowing for restart or upgrade but at the cost
- *   of forbidding storage of references to ephemeral data.  Defaults to false.
- * @property {boolean} [fakeDurable=false]  This store pretends to be a durable store
- *   but does not enforce that the things stored in it actually be themselves
- *   durable (whereas an actual durable store would forbid storage of such
- *   items).  This is in service of allowing incremental transition to use of
- *   durable stores, to enable normal operation and testing when some stuff
- *   intended to eventually be durable has not yet been made durable.  A store
+ * @typedef {object} StoreOptions Of the dimensions on which KeyedStores can
```

Comment:
Sorry for sending you off on this chase. When I compare before and after in WebStorm, the tool seems to have improved its parsing, and both the old and new render just fine. If you want to leave these formatting changes as they were, that would be fine.

Reply:
haha. well it's better enough that I'll click teh green button